students/scotchwsplenda/Lesson_9/mailroom_obl_oo/mailroom_oo.py

```diff
@@ -0,0 +1,63 @@
+from operator import itemgetter
+
+
+class Donor():
+
+    def __init__(self, name=None, donations=[]):
+        self.name = name
+        self.donations = donations
+
+    def thanks_mail(self, name, amount):
+        email = ''.join((
+            "\nDear {},\n",
+            "Thanks for the ${:.2f} bucks.\n",
+            "We'll spend it real good baby.\n"
+            )).format(name, amount)
+        return email
```

Comment:
a nice addition here would be to provide methods/properties for average/total/num of values.